Makefile.toml

```diff
@@ -262,6 +264,57 @@ cargo build \
 '''
 ]
 
+[tasks.build-kmod-kit]
+dependencies = ["build-packages"]
+script = [
+'''
+mkdir -p "${BUILDSYS_ARCHIVES_DIR}"
+kmod_kit="${BUILDSYS_VARIANT}-${BUILDSYS_ARCH}-kmod-kit-v${BUILDSYS_VERSION_IMAGE}"
+rm -f "${BUILDSYS_ARCHIVES_DIR}/${kmod_kit}.tar.xz"
+
+prepare_kmod_kit="
+set -e -o pipefail
+
+mkdir /tmp/${kmod_kit}
+
+# Retrieve the toolchain and kernel archives.
+curl --silent --output /tmp/${kmod_kit}/toolchain.tar.xz \
+  https://${BUILDSYS_SDK_SITE}/${BUILDSYS_TOOLCHAIN}.tar.xz
+find ./rpms -name '*-${BUILDSYS_ARCH}-kernel-archive-*.rpm' \
+  -exec rpm2cpio {} \; | cpio -idmu --quiet
```

Comment:
This brings to mind times I've been bitten by old rpms... should we be a little more selective somehow?  Or just take the latest?

Reply:
That's a good call out - I've been testing on a clean tree, but the result of this would be ambiguous if two kernel-archive packages were present, which could happen when testing an update to a newer kernel.